UnitsNet.WindowsRuntimeComponent/GeneratedCode/Quantity.g.cs

```diff
@@ -30,6 +31,665 @@ namespace UnitsNet
     /// </summary>
     internal static partial class Quantity
     {
+        public static readonly IDictionary<string, QuantityInfo> ByName = new Dictionary<string, QuantityInfo>
```

Comment:
This is not obvious, but UnitsNet.WindowsRuntimeComponent is deprecated: #815 

I see the [AppVeyor build failed](https://ci.appveyor.com/project/angularsen/unitsnet/builds/36676346), so it is probably best to revert and not touch anything under that namespace in order to avoid problems. WRC is currently receiving new quantities and units via changes to JSON files, but we stopped maintaining its code generator some time ago.

Reply:
done, reverted